targets/ingester/src/transform/agreements/index.ts

```diff
@@ -54,6 +54,7 @@ export default async function getAgreementDocuments() {
           agreement.shortTitle
         }`,
         is_searchable: true,
+        is_published: agreement.active,
```

Comment:
j'aurais fait : 

```suggestion
        is_published: !!agreement.active,
``` 
pour ne pas avoir d'`undefined` si on oublie de le renseigner

Reply:
Si il est undefined, il ne sera tout simplement pas mis à jour en BDD. Je pensais que ce comportement était plus logique, non ?